universal-login-sdk/test/services/FutureWalletFactory.ts

```diff
@@ -0,0 +1,47 @@
+import {expect} from 'chai';
+import {utils, Wallet} from 'ethers';
+import {createMockProvider, getWallets, deployContract} from 'ethereum-waffle';
+import {getContractWhiteList} from '@universal-login/relayer';
+import {ETHER_NATIVE_TOKEN} from '@universal-login/commons';
+import {getDeployData} from '@universal-login/contracts';
+import WalletMaster from '@universal-login/contracts/build/WalletMaster.json';
+import Factory from '@universal-login/contracts/build/ProxyCounterfactualFactory.json';
+import ProxyContract from '@universal-login/contracts/build/Proxy.json';
+import {FutureWalletFactory} from '../../lib/services/FutureWalletFactory';
+import {BlockchainService} from '../../lib/services/BlockchainService';
+
+describe('INT: FutureWalletFactory', async () => {
+  let wallet: Wallet;
+  let futureWalletFactory: FutureWalletFactory;
+
+  before(async () => {
+    const provider = createMockProvider();
```

Comment:
Use `setupSDK` instead.
https://github.com/UniversalLogin/UniversalLoginSDK/blob/86a882e377f95d2f8ff2b3a6b10491c72f05986d/universal-login-sdk/lib/SetupSdk.ts

Note:
It will create relayer, that already has the whole config and deploy the whole setup.